sessionctx/binloginfo/binloginfo.go

```diff
@@ -89,6 +89,10 @@ func (info *BinlogInfo) WriteBinlog(clusterID uint64) error {
 		if err == nil {
 			return nil
 		}
+		if strings.Contains(err.Error(), "received message larger than max") {
```

Comment:
Should we return a more friendly error message here? Such as "too large binlog xxxx".